core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -348,7 +347,7 @@ public synchronized InetSocketAddress getAddress() throws UnavailableException {
       try {
         return rpc.call();
       } catch (StatusRuntimeException e) {
-        AlluxioStatusException se = GrpcExceptionUtils.fromGrpcStatusException(e);
+        AlluxioStatusException se = AlluxioStatusException.fromStatusRuntimeException(e);
         if (se.getStatus() == Status.UNAVAILABLE
```

Comment:
@ggezer Why do we only convert a few types of exceptions? Why not convert all `StatusRuntimeException`s to `AlluxioStatusException`s?

Reply:
We convert all, but don't throw immediately for the listed few.

Reply:
ah you're right, got it